File: mynote-android/app/src/main/java/com/xuyazhou/mynote/model/db/UserSetting.java
package com.xuyazhou.mynote.model.db;

import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

/**
 * Author: lampard_xu(xuyazhou18@gmail.com)
 * *
 * Date: 2016-12-19
 */
@Table(database = AppDatabase.class)
public class UserSetting extends BaseModel {

    @PrimaryKey(autoincrement = true)
    @Column
    long _id;


    @Column
    long cratedTime;

    @Column
    long modifiedTime;

    @Column
    String lockType;

    @Column
    boolean _deleted;

    @Column
    boolean isLogin;


    @Column
    String listType;


    public boolean isLogin() {
        return isLogin;
    }

    public void setLogin(boolean login) {
        isLogin = login;
    }

    public long get_id() {
        return _id;
    }

    public void set_id(long _id) {
        this._id = _id;
    }

    public long getCratedTime() {
        return cratedTime;
    }

    public void setCratedTime(long cratedTime) {
        this.cratedTime = cratedTime;
    }

    public long getModifiedTime() {
        return modifiedTime;
    }

    public void setModifiedTime(long modifiedTime) {
        this.modifiedTime = modifiedTime;
    }

    public String getListType() {
        return listType;
    }

    public void setListType(String listType) {
        this.listType = listType;
    }

    public boolean is_deleted() {
        return _deleted;
    }

    public void set_deleted(boolean _deleted) {
        this._deleted = _deleted;
    }
}

File: mynote-ios/MyNote-Ios/Models/db/UserSetting.swift
//
//  UserSetting.swift
//  MyNote-Ios
//
//  Created by xuyazhou on 2017/11/29.
//  Copyright © 2017年 xuyazhou. All rights reserved.
//

import RealmSwift

class UserSetting: Object {
    dynamic var id = 0
    dynamic var lockType = ""
    dynamic var createTime = 0
    dynamic var modifiedTime = 0
    dynamic var deleted = false
    dynamic var isLogin = false
    dynamic var listType = ""
    dynamic var isHaveNew = false
    dynamic var isHavedelete = false
    

    override static func primaryKey() -> String? {
        return "id"
    }
}
